specification/consumption/resource-manager/readme.md

```diff
@@ -26,7 +26,17 @@ These are the global settings for the Consumption API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2021-10
+tag: package-2022-06
+```
+
+### Tag: package-2022-06
+
+These settings apply only when `--tag=package-2022-06` is specified on the command line.
+
+```yaml $(tag) == 'package-2022-06'
+input-file:
+  - Microsoft.Consumption/stable/2022-06-01/consumption.pricesheets.operations.json
+  - Microsoft.Consumption/stable/2021-10-01/consumption.json
```

Comment:
nit

Can you move "Microsoft.Consumption/stable/2022-06-01/consumption.pricesheets.operations.json" after "Microsoft.Consumption/stable/2021-10-01/consumption.json"

As `PriceSheet` conflict with `PriceSheets` in Java, I'd like "consumption.json" take precedence, to avoid breaking changes.

Reply:
@jayati-tehri , can you make above change?

Reply:
I have reordered the files in readme

Reply:
Thank you very much. This indeed avoid unnecessary break in Java SDK. https://github.com/azure-sdk/azure-sdk-for-java/pull/3576/files#diff-0dae5b9adaff9878f130577ec4e2206c49534fbf67b66635688c523751e2ae85